api/src/lib.rs

```diff
@@ -244,6 +245,22 @@ impl Client {
         self.delete(self.endpoints.source_by_id(&source_id)?)
     }
 
+    /// Set a quota
+    pub fn create_quota(
```

Comment:
Agreed it would be very useful to have a `get` command as well. Ideally it would show you which quotas were defaults and which had been changed, but this requires some backend changes and can be a future enhancement.